controller/volume_attachment_controller.go

```diff
@@ -91,6 +92,12 @@ func NewLonghornVolumeAttachmentController(
 	}, 0)
 	vac.cacheSyncs = append(vac.cacheSyncs, ds.EngineInformer.HasSynced)
 
+	ds.KubeNodeInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
Why need to watch delete event?

Why not using `AddEventHandlerWithResyncPeriod`?

Reply:
Fixed, thanks!